apps/microtvm/pyproject.toml

```diff
@@ -93,7 +93,7 @@ cffi = {version = "^1.14", optional = true}
 mxnet = {version = "^1.6.0", optional = true}
 
 # ONNX frontend
-onnx = {version = "1.6.0", optional = true}
+onnx = {version = "1.6.0", optional = false}
```

Comment:
I wonder if we can instead change `apps/microtvm/reference-vm/zephyr/setup.sh` to do `pip install -E importer-onnx`? I'd like to eventually unify this file with the one produced by https://discuss.tvm.apache.org/t/rfc-python-dependencies-in-tvm-ci-containers/9011

Reply:
@areusch Thanks for the review. I moved it to "setup.sh".